samples/v1p1beta1/deleteNotificationConfig.js

```diff
@@ -0,0 +1,38 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+'use strict';
+
+function main(organizationId = 'your-org-id', configId = 'your-config-id') {
+  // [START scc_delete_notification_config]
+  // npm install @google-cloud/security-center/
+  const {SecurityCenterClient} =
+      require('@google-cloud/security-center').v1p1beta1;
+
+  const client = new SecurityCenterClient();
+
+  // TODO(UpdateMe) organizationId = "your-org-id";
+  // TODO(UpdateMe) configId = "your-config-id";
+  const formattedConfigName =
+      client.notificationConfigPath(organizationId, configId);
+
+  async function deleteNotificationConfg() {
+    client.deleteNotificationConfig({name: formattedConfigName});
```

Comment:
is this an async function?  Guessing we need to `await` here

Reply:
Done.